{
    "config": {
        "abort": {
            "single_instance_allowed": "Jo m\u00e4\u00e4ritetty. Vain yksi konfiguraatio mahdollinen."
        },
        "error": {
            "cannot_connect": "Yhteyden muodostaminen ep\u00e4onnistui",
            "unknown": "Odottamaton virhe"
        },
        "step": {
            "user": {
                "data": {
                    "host": "Palvelin",
                    "password": "Salasana",
                    "username": "K\u00e4ytt\u00e4j\u00e4tunnus"
                },
                "data_description": {
                    "host": "Lutronin p\u00e4\u00e4toistimen is\u00e4nt\u00e4nimi tai IP-osoite."
                },
                "description": "Sy\u00f6t\u00e4 p\u00e4\u00e4toistimen kirjautumistiedot",
                "title": "P\u00e4\u00e4toistimen asetukset"
            }
        }
    },
    "issues": {
        "deprecated_yaml_import_issue_cannot_connect": {
            "description": "Lutronin konfigurointi YAML:n avulla poistetaan, mutta YAML-konfiguraation tuonnissa tapahtui yhteysvirhe.\n\nAsioita, joita voit kokeilla:\nVarmista, ett\u00e4 kotiavustajasi voi tavoittaa p\u00e4\u00e4toistimen.\nK\u00e4ynnist\u00e4 p\u00e4\u00e4toistin uudelleen irrottamalla se 60 sekunnin ajaksi.\nYrit\u00e4 kirjautua p\u00e4\u00e4toistimeen selaimella m\u00e4\u00e4ritt\u00e4m\u00e4ll\u00e4si IP-osoitteella ja samoilla kirjautumistiedoilla.\n\nK\u00e4ynnist\u00e4 sitten Home Assistant uudelleen ja yrit\u00e4 tuoda t\u00e4m\u00e4 integraatio uudelleen.\n\nVaihtoehtoisesti voit poistaa Lutron-konfiguraation kokonaan YAML-konfiguraatiostasi, k\u00e4ynnist\u00e4\u00e4 Home Assistantin uudelleen ja lis\u00e4t\u00e4 Lutron-integraation manuaalisesti.",
            "title": "Lutronin YAML-konfiguraation tuonti ei pysty muodostamaan yhteytt\u00e4 palvelimeen"
        },
        "deprecated_yaml_import_issue_unknown": {
            "description": "Lutronin konfigurointi YAML:n avulla poistetaan, mutta olemassa olevan konfiguraation tuonnin yhteydess\u00e4 tapahtui tuntematon virhe.\nAsennus ei jatku.\n\nErityinen virhe l\u00f6ytyy lokitiedostoista. Todenn\u00e4k\u00f6isin syy on verkkovirhe tai p\u00e4\u00e4toistin on poissa k\u00e4yt\u00f6st\u00e4 tai sen konfiguraatio on virheellinen.\n\nVarmista, ett\u00e4 Lutron-j\u00e4rjestelm\u00e4 toimii oikein, ja k\u00e4ynnist\u00e4 Home Assistant uudelleen yritt\u00e4\u00e4ksesi tuontia uudelleen.\n\nVaihtoehtoisesti voit poistaa Lutron-konfiguraation kokonaan YAML-konfiguraatiostasi, k\u00e4ynnist\u00e4\u00e4 Home Assistantin uudelleen ja lis\u00e4t\u00e4 Lutron-integraation manuaalisesti.",
            "title": "Lutronin YAML-konfiguraation tuontipyynt\u00f6 ep\u00e4onnistui tuntemattoman virheen vuoksi."
        }
    }
}